<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connectWin</class>
 <widget class="QDialog" name="connectWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connexion AvatarBattle2D</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>381</width>
     <height>78</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>IP Serveur</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="serveurIP">
      <property name="text">
       <string>127.0.0.1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Port Serveur</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="serveurPort">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>65000</number>
      </property>
      <property name="value">
       <number>50885</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Connexion Ã  un serveur AvatarBattle2D&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>212</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="boutonConnexion">
      <property name="text">
       <string>Connexion</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="boutonDeconnexion">
      <property name="text">
       <string>Deconnexion</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>381</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QTextEdit" name="messageList">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 217, 217);
background-color: rgb(0, 0, 0);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="boutonFermer">
      <property name="text">
       <string>Fermer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
